mmv1/third_party/terraform/tests/resource_gke_hub_feature_membership_test.go.erb

```diff
@@ -615,15 +626,15 @@ func TestAccGkeHubFeatureMembership_gkehubFeatureMesh(t *testing.T) {
 	})
 }
 
-func testAccGkeHubFeatureMembership_gkehubFeatureMeshStart(context map[string]interface{}) string {
+func testAccGkeHubFeatureMembership_meshStart(context map[string]interface{}) string {
 	return gkeHubFeatureProjectSetup(context) + Nprintf(`
 resource "google_container_cluster" "primary" {
   project = google_project.project.project_id
   name               = "tf-test-cl%{random_suffix}"
   location           = "us-central1-a"
   initial_node_count = 1
   provider = google-beta
-  depends_on = [google_project_service.mci, google_project_service.container, google_project_service.container, google_project_service.gkehub, google_project_service.mesh]
```

Comment:
Why is google_project_service.mci no longer a prerequisite?

Reply:
mci is not a prerequisite or a dependency for the mesh feature, so I've removed it.